src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -48,387 +102,45 @@ describe('convertInclude', () => {
     process.env.SIMORGH_INCLUDES_BASE_AMP_URL = initialIncludesAmpBaseUrl;
   });
 
-  it('should fetch and convert an include block to an idt1 block', async () => {
-    fetch.mockResponse(() => Promise.resolve(idt1Markup));
-    const input = idt1Block;
-    const expected = {
-      type: 'include',
-      model: {
-        href: '/indepthtoolkit/quizzes/123-456',
-        index: 0,
-        isAmpSupported: false,
-        type: 'idt1',
-        html: idt1Markup,
-      },
-    };
-    expect(
-      await convertInclude(input, pageData, null, canonicalPathname),
-    ).toEqual(expected);
-    expect(fetch).toHaveBeenCalledTimes(1);
-    expect(fetch).toHaveBeenCalledWith(
-      'https://foobar.com/includes/indepthtoolkit/quizzes/123-456',
-      {
-        timeout: 3000,
-      },
-    );
-    expect(loggerMock.error).not.toHaveBeenCalled();
-    expect(loggerMock.info).toHaveBeenCalledTimes(1);
-    expect(loggerMock.info).toHaveBeenCalledWith(INCLUDE_REQUEST_RECEIVED, {
-      url: 'https://foobar.com/includes/indepthtoolkit/quizzes/123-456',
-    });
-  });
-
-  it('should fetch and convert an include block to an idt2 block for a canonical request', async () => {
-    fetch.mockResponse(() => Promise.resolve(idt2Markup));
-    const input = idt2Block;
-    const expected = {
-      type: 'include',
-      model: {
-        href: '/idt2/111-222-333-444-555',
-        index: 1,
-        isAmpSupported: true,
-        type: 'idt2',
-        html: idt2Markup,
-        imageBlock: {
-          alt: 'image alt text',
-          height: 1864,
-          layout: 'responsive',
-          src: 'https://foobar.com/includes/idt2/111-222-333-444-555/image/816',
-          srcset:
-            'https://foobar.com/includes/idt2/111-222-333-444-555/image/470 470w,https://foobar.com/includes/idt2/111-222-333-444-555/image/816 816w',
-          width: 1632,
-        },
-      },
-    };
-    expect(
-      await convertInclude(input, pageData, null, canonicalPathname),
-    ).toEqual(expected);
-    expect(fetch).toHaveBeenCalledTimes(1);
-    expect(fetch).toHaveBeenCalledWith(
-      'https://foobar.com/includes/idt2/111-222-333-444-555/html',
-      {
-        timeout: 3000,
-      },
-    );
-    expect(loggerMock.error).not.toHaveBeenCalled();
-    expect(loggerMock.info).toHaveBeenCalledTimes(1);
-    expect(loggerMock.info).toHaveBeenCalledWith(INCLUDE_REQUEST_RECEIVED, {
-      url: 'https://foobar.com/includes/idt2/111-222-333-444-555/html',
+  describe('Data dependent conditions', () => {
```

Comment:
From a readability perspective, I think I'd favour all these data dependent tests having their data in an `it.each` table rather than in the array at the top; the ones with `runAdditionalExpectations` could just be seperate tests.